src/index.css

```
@@ -164,3 +164,34 @@ textarea:disabled {
 .code::after {
   content: "";
 }
+
+.hovertext::before {
```

```suggestion
.hover-text::before {
```
css 클래스는 일관성을 위해 케밥 케이스로 수정하면 좋을 것 같습니다.

---

수정했습니다!